docs/resources/azure_integration_attachment.md

```diff
@@ -0,0 +1,59 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spacelift_azure_integration_attachment Resource - terraform-provider-spacelift"
+subcategory: ""
+description: |-
+  spacelift_azure_integration_attachment represents the attachment between a reusable Azure integration and a single stack or module.
+---
+
+# spacelift_azure_integration_attachment (Resource)
+
+`spacelift_azure_integration_attachment` represents the attachment between a reusable Azure integration and a single stack or module.
+
+## Example Usage
+
+```terraform
+# For a stack
+resource "spacelift_azure_integration_attachment" "readonly" {
+  integration_id  = spacelift_azure_integration_attachment.example.id
```

Comment:
I think the resource here should be `spacelift_azure_integration` rather than `spacelift_azure_integration_attachment`. Same with the module example.